capstone/capdb/migrations/0013_auto_20171211_2010.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-12-11 20:10
+from __future__ import unicode_literals
```

Comment:
I think there might be a merge conflict because there already is a 0013_migration